rust/private/rustdoc.bzl

```diff
@@ -37,6 +37,7 @@ def _strip_crate_info_output(crate_info):
         aliases = crate_info.aliases,
         # This crate info should have no output
         output = None,
+        metadata = crate_info.metadata,
```

Comment:
Given the comment 2 lines above, probably this crate info should also not have a metadata output?

Reply:
Done.